packages/browsers/src/CLI.ts

```diff
@@ -261,50 +264,21 @@ export class CLI {
         },
         async argv => {
           const args = argv as unknown as InstallArgs;
-          args.platform ??= detectBrowserPlatform();
-          if (!args.platform) {
-            throw new Error(`Could not resolve the current platform`);
-          }
-          if (args.browser.buildId === 'pinned') {
-            const pinnedVersion = this.#pinnedBrowsers?.[args.browser.name];
-            if (!pinnedVersion) {
-              throw new Error(
-                `No pinned version found for ${args.browser.name}`
-              );
-            }
-            args.browser.buildId = pinnedVersion;
+          if (this.#pinnedBrowsers && !args.browser) {
+            await Promise.all(
```

Comment:
should we use Promise.allSettled here and throw if there is any error?

Reply:
I think it make sense, one problem with the updated implementation is surfacing the error messages from the install function.

Reply:
I think re-throwing the first one would result in the same behavior w.r.t to errors